met-api/src/met_api/services/email_verification_service.py

```diff
@@ -106,7 +108,8 @@ def _send_verification_email(email_verification: dict) -> None:
             raise ValueError('Engagement not found')
 
         subject, body, args, template_id = EmailVerificationService._render_email_template(
-            survey, email_verification.get('verification_token'), email_verification.get('type'), participant_id)
+            survey, email_verification.get('verification_token'),
```

Comment:
may we please prettify the style here, one param in each line maybe

Reply:
updated the style, thanks.